.drone.yml

```diff
@@ -107,7 +107,7 @@ steps:
   commands:
   - R -s -e 'devtools::install_deps(upgrade = '"'"'never'"'"')'
   # nmrec isn't available until the 2023-09-19 snapshot.
-  - git clone --branch 0.3.0.8001 --depth 1 https://github.com/metrumresearchgroup/nmrec.git /tmp/nmrec
+  - git clone --branch 0.4.0 --depth 1 https://github.com/metrumresearchgroup/nmrec.git /tmp/nmrec
```

Comment:
I think this is okay, but as a side remark: in the spirit of the "oldest" build, I suppose it might make sense to eventually downgrade this to the nmrec version that's on the 2023-09-19 snapshot

Reply:
I could go either way on this. I don't think it practically matters at this point, and that would make it skip some tests. So I think I'd opt for keeping it this way for now.

Reply:
> I could go either way on this. I don't think it practically matters at this point, and that would make it skip some tests.

That's true, the from my standpoint that's a good thing.  The other runs already cover this, and this run would tell you that you set up your skips properly so that your tests work with an older nmrec.

I agree that's not super valuable info, especially at this point.

>  So I think I'd opt for keeping it this way for now.

Sure, I agree there.  I should of probably rephrased my comment to make it really clear I wasn't hoping for this PR to change this.  (Holding off on approving as I do a scorecard run.)